gwu-project-1/script.js

```diff
@@ -0,0 +1,15 @@
+let inputText = document.querySelector("#input-text");
```

Comment:
Is there a a reason why this is `let` instead of `const`?

Reply:
No, I have no specific reason. I have re-declared the variables.